docs/v0.5.x/writing-a-plugin.md

```diff
@@ -76,6 +76,25 @@ module.exports = {
 
 That's seriously about as difficult as it gets. However, read on for some more advanced info to get the most out of your ember-cli-deploy plugin.
 
+### Configuration naming
+By convention, plugin configuration should be kept in `config/deploy.js` and scoped by
+the plugin's name.
+
+E.g. for an `ember-cli-deploy-example` plugin with `name: 'example'`, the configuration might look like:
+
+```javascript
+// config/deploy.js
+module.exports = return function(environment) {
+  return {
+    "example": {
+      bucket: "my-app-" + environment,
+      awsAccessKeyId: process.env.AWS_ACCESS_KEY_ID,
+      awsSecretAccessKey: process.env.AWS_SECRET_ACCESS_KEY
+    }
+  };
+};
+```
```

Comment:
the idea is that if they don't use the `BasePlugin` and instead they implement their own `configure` hook they should make sure they scope their config under a `key` named as their plugin.

at the same time  even in case of using the `BasePlugin` I didn't feel it was clear how the config  was being looked up in the main config object

that said, if you guys think it is I'm all in for killing this :)